internal/core/storer/etcd.go

```diff
@@ -138,16 +138,23 @@ func (s *EtcdV3) Create(ctx context.Context, key, value string) error {
 }
 
 // Update a value at the specified key
-func (s *EtcdV3) Update(ctx context.Context, key, value string) error {
+func (s *EtcdV3) Update(ctx context.Context, key, value string, version int64) error {
 	ctx, cancel := context.WithTimeout(ctx, s.timeout)
 	defer cancel()
 
-	_, err := s.client.Put(ctx, key, value)
+	txn := s.client.Txn(ctx).
+		If(clientv3.Compare(clientv3.Version(key), "=", version)).
+		Then(clientv3.OpPut(key, value))
+
+	resp, err := txn.Commit()
 	if err != nil {
 		log.Errorf("etcd update key[%s] failed: %s", key, err)
 		return fmt.Errorf("etcd update key[%s] failed: %s", key, err)
 	}
-
+	if !resp.Succeeded {
+		log.Infof("etcd update key[%s] failed", key)
```

Comment:
Would be better to avoid the word "failed" as this result is expected.

Reply:
> Would be better to avoid the word "failed" as this result is expected.

Wait me change